### DISCUSSION TITLE
Translate the Admin UI

### DISCUSSION THREAD

**Discussion by tsvetann** [medusajs/medusa#4120] [Feature Requests]
Hi,

I just wanted to see if making the Admin UI multilingual is in the roadmap. If not, are there any intentions to include it?

Thanks

**Comment by mj-mehdizadeh** [comment]
any update?

**Comment by langovoi** [comment]
PR which provides translation for admin UI: https://github.com/medusajs/medusa/pull/4190

**Reply by tsvetann** [reply]
the PR is cancelled

**Reply by langovoi** [reply]
But it could be used as start point, because a lot of done there.
